core/templates/pages/collection-editor-page/collection-editor-page-auth.guard.ts

```diff
@@ -51,7 +51,7 @@ export class CollectionEditorPageAuthGuard implements CanActivate {
           resolve(true);
         })
         .catch(err => {
-          let statusCode = err ? err.error.status_code : 401;
+          let statusCode = err ? err.status : 401;
```

Comment:
Is this intended? also use const instead of let

Reply:
Yes, this is intended. This was a typo so that's why I changed it. Done!!